### ISSUE TITLE
resident evil 5 steamworks

### ISSUE THREAD

**Issue by alfredox123** [lucasassislar/nucleuscoop#1]
will this new version work with resident evil 5 steamworks version?

**Comment by HYPERTiZ** [comment]
Why would it matter if RE5 non Gold has the ability to natively play splitscreen?

**Comment by alfredox123** [comment]
because the non gold doesnt have the dlc to play coop on same pc, and will be interesting to maybe play versus on splitscreen too

**Comment by HYPERTiZ** [comment]
Thats weird: i dont have a single dlc installed nor the 'untold story' one.

No is it gold..

**Comment by HYPERTiZ** [comment]
Nor*

**Comment by alfredox123** [comment]
you have to buy the dlc to have it on steam.....

**Comment by HYPERTiZ** [comment]
Let me see.. In dlcs section of my pc since it does in game have import gfwl save

**Comment by alfredox123** [comment]
http://store.steampowered.com/app/352000/

**Comment by HYPERTiZ** [comment]
I literally dont have that dlc...

**Comment by alfredox123** [comment]
yeah, you have to buy it to have it, like any other paid dlc on any other game...

that dlc released like 8 month ago for steam re5

**Comment by HYPERTiZ** [comment]
Wel awkwardly im an idiot guess i cant. Also got dlc; still cant. ??

**Comment by HYPERTiZ** [comment]
Apparently it likes to (instant black window in game) close itself... Hmm

**Comment by HYPERTiZ** [comment]
right... i went back to try it; im wrong.. apparently it only works with gfwl :(

**Comment by XxRaPiDK3LLERxX** [comment]
CAPCOM removed the splitscreen code that was in GFWL version...

**Comment by alfredox123** [comment]
no, you can still play splitscreen with gfwl version

**Comment by alfredox123** [comment]
so... is gonna be supported?
